src/color/color_utils.js

```diff
@@ -79,14 +81,15 @@ define(function(require) {
   /**
    * For a color expressed as an RGBA array, return the corresponding HSBA value
    * 
-   * @param {Array} rgba An 'array' object that represents a list of
-   *                          RGB colors on a scale of 0-255
-   * @return {Array} an array of HSB values, on a scale of 0-255
+   * @param {Array} rgba An 'array' object that represents a list of RGB colors
+   * @param {Array} maxes An 'array' object that represents the RGB range maxes
+   * @return {Array} an array of HSB values, scaled by the HSB-space maxes
    */
-  p5.ColorUtils.rgbaToHSBA = function(rgba) {
-    var var_R = rgba[0]/255;
-    var var_G = rgba[1]/255;
-    var var_B = rgba[2]/255;
+  p5.ColorUtils.rgbaToHSBA = function(rgba, maxes) {
+    var var_R = rgba[0]/maxes[0];
+    var var_G = rgba[1]/maxes[1];
+    var var_B = rgba[2]/maxes[2];
+    var var_A = rgba[3]/maxes[3];
```

Comment:
Rockin'. 

​
I think that would be really helpful.  I'll see what I can do. 


Input is always welcome. I'll figure out some way to get it up somewhere collaborative as soon as I can.